docs/spec/staking/01_state.md

```diff
@@ -24,6 +24,17 @@ type Params struct {
 
 ## Validator
 
+Validators can have one of three statuses
+
+- `unbonded`: The validator is not in the active set. They cannot sign blocks and do not earn
+rewards. They can receive delegations. Unbonding from an `unbonded` validator is immediate.
+- `bonded`: The validator has sufficient bonded tokens to join the active set. They are signing
+blocks and receiving rewards. They can receive further delegations. They can be slashed for
+misbehavior. Delegators to this validator who unbond their delegation must wait the duration of the
+UnbondingTime, a chain-specific param.
+- `unbonding`: The validator has initiated an unbonding of their self delegation. They must wait the
```

Comment:
There are other means in which a validator can start to unbond (e.g. being kicked out of the active set).